lib/cli/kit/system.rb

```diff
@@ -191,9 +191,16 @@ def resolve_path(a, env)
           # If only one argument was provided, make sure it's interpreted by a shell.
           return ["true ; " + a[0]] if a.size == 1
           return a if a.first.include?('/')
-          item = env.fetch('PATH', '').split(':').detect do |f|
-            File.exist?("#{f}/#{a.first}")
+
+          # If executing "foo", and no foo executable was found on the PATH, shells will
+          # see if one exists in the current directory and execute that.
+          paths = env.fetch('PATH', '').split(':')
```

Comment:
I feel like we should considering pruning empty or `.` (or even `./*`?) from whatever the `PATH` is. Might be ok for someone to have that for their interactive shell, but I can't see why it’d be valid within a `CLI::Kit` context

Reply:
I don't think we should do that, for a couple of reasons:
1. This method is meant to resolve commands similar to shells, so pruning could be surprising.
2. I don't think cli-kit should be prescriptive about that. I think that would make sense at a higher level (i.e., whatever is using cli-kit)